src/lib/trusted-types.ts

```diff
@@ -0,0 +1,43 @@
+function getTrustedTypes() {
+  // tslint:disable-next-line
+  return (window as any).TrustedTypes;
```

Comment:
Nit: Can we have window.trustedTypes || window.TrustedTypes here? We're renaming the main entry point for the API soon (https://bugs.chromium.org/p/chromium/issues/detail?id=991944)

Reply:
Yeah.